TACOptimizations/ControlFlowGraph.cs

```diff
@@ -22,6 +22,9 @@ public class ControlFlowGraph
         private List<int> _dfn;
         public IReadOnlyList<int> DepthFirstNumeration => _dfn.AsReadOnly();
 
+        private List<(int, int, EdgeType)> _classifiedEdges;
+        public IReadOnlyList<(int, int, EdgeType)> ClassifiedEdges => _classifiedEdges.AsReadOnly();
```

Comment:
`AsReadOnly()` можно убрать, если указываете явно тип `IReadOnlyList`. Я проглядел это, когда добавлял другие такие же, так что уберите все `AsReadOnly()` в этом файле, если не сложно.
```suggestion
        public IReadOnlyList<(int, int, EdgeType)> ClassifiedEdges => _classifiedEdges;
```

Reply:
Выпилил